### PR TITLE
Element Hiding - Fix forEach loop HTMLCollection regression

### PR DESCRIPTION
**Pull request by dharb**
**Reviewer:** @kdzwinel 

## Description:
I introduced a bug in my last commit in https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/281 by calling forEach on an HTMLCollection. I'm guessing I forgot to enable/disable the extension after reloading it to clear cached content scripts after making the changes in that commit and missed the errors. This PR just fixes that issue by using Array.prototype.forEach.call in order to access the forEach method.

## Steps to test this PR:
See https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/281

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [x] Get advice or leverage existing code
- [x] Agree on technical approach with reviewer (if the changes are nuanced)
- [x] Ensure that there is a testing strategy (and documented non-automated tests)
- [x] Ensure there is a documented monitoring strategy (if necessary)
- [x] Consider systems implications

### CODE REVIEW

**Comment by dharb** [comment]
Hey @kdzwinel , mind giving this a quick look and merge?

File: shared/js/content-scripts/element-hiding.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
         constructor () {
             // Determine if content script is running in iframe or main frame
             this.frameType = window === window.top ? 'main' : window.parent === window.top ? 'topLevelFrame' : 'nestedFrame'
+            this.foundFrames = []
```
Comment (kdzwinel) [inline shared/js/content-scripts/element-hiding.js]: is there any good reason to keep those frames around?

File: shared/js/content-scripts/element-hiding.js [hunk 2]
```diff
@@ -45,7 +46,8 @@
                     window.top.postMessage({frameUrl: document.location.href, type: 'frameIdRequest'}, req.mainFrameUrl)
                 }
             } else if (req.type === 'blockedFrame') {
-                document.getElementsByTagName('iframe').forEach((frame) => {
+                this.foundFrames = document.getElementsByTagName('iframe')
+                Array.prototype.forEach.call(this.foundFrames, (frame) => {
```
Comment (kdzwinel) [inline shared/js/content-scripts/element-hiding.js]: how about we replace `getElementsByTagName` with `querySelectorAll`? That way we will be able to use `.forEach`.

**Comment by kdzwinel** [comment]
Someone on twitter pointed out that typescript would have caught that for us. Food for though!

**Comment by kdzwinel** [comment]
Looks, and works, great 👍

**Pull request merged**